.github/workflows/ci.yml

```diff
@@ -33,15 +35,18 @@ jobs:
           - rhel8-x64
           - al2-x64
     steps:
-        # We can't use the `uses: docker://image` version yet, GitHub lacks authentication for actions -> packages
+    - uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ env.CRT_CI_ROLE }}
+        aws-region: ${{ env.AWS_DEFAULT_REGION }}
```

Comment:
you'd mentioned in-office having some shared code in aws-crt-builder

did you decide to just copy/paste the same 4 lines instead of doing that?

I'm ok with either, just asking